src/app/retrospective/sprint-detail/sprint-detail.component.html

```diff
@@ -86,6 +86,9 @@ <h4 class="title">
                             <p>
                                 Sprint Points: <b>{{ sprintBugs.TotalPointsEarned | number:decimalFormat }}</b>
                             </p>
+                            <p *ngIf="sprintBugs.ActualHours">
+                                Actual Hours: <b>{{  sprintBugs.ActualHours / 60 | number:decimalFormat}}</b>
```

Comment:
Make `60` a constant if not already and use it.

Reply:
done